MonitoringSystem/Client/MonitoringReporter.py

```diff
@@ -0,0 +1,133 @@
+"""
+This class is used to insert data to elasticsearch. It uses an internal list which is used to keep messages in the memory. 
```

Comment:
yes. this is a general class as you said it inserts everywhere if the put provided. I improve the document.